File: generator/output/entityModel/PlatformModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Platform
*/
@Data()
public class PlatformModel {
  public Long id = null;

  /**
  *
  */
  public Integer deleted = null;

  /**
  *
  */
  public String name = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/platform.model.ts
class PlatformModel {
  public id: number = 0;

  // deleted 
  public deleted?: number = 0;

  // name 
  public name: string;

  public created_at: string;

  public updated_at: string;
}

const PlatformModelFields = {
  id: "id",
  deleted: "deleted",
  name: "name",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {PlatformModel, PlatformModelFields};
